packages/carbon-graphs/src/js/helpers/axis.js

```diff
@@ -1197,7 +1197,6 @@ const calculateAxesLabelSize = (config) => {
   config.axisLabelHeights.x = 0;
   config.axisLabelWidths.y = 0;
   config.axisLabelWidths.y2 = 0;
-  config.axisInfoRowLabelHeight = 0;
```

Comment:
Can you please explain the reason for this removal?

Reply:
This was interfering with the new logic to update the container height to accommodate for x-axis and and axis info row on a bar chart. Removing this resolved the issue and caused all unit tests to pass again.